lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1367,7 +1370,7 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit', 'embed' ),
 					'arg_options' => array(
-						'sanitize_callback' => 'sanitize_title',
+						'sanitize_callback' => 'sanitize_key',
```

Comment:
We need to use `sanitize_title()` because core uses `sanitize_title()` for the slug, not `sanitize_key()`. See #1585